tests/Fixtures/Integration/priority/no_empty_statement,multiline_whitespace_before_semicolons.test

```diff
@@ -0,0 +1,21 @@
+--TEST--
```

Comment:
duplicates https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3436/files

Reply:
^_^
:+1: 
I remove it later, I was just going through the missing ones in arbitrary order without checking open PR's :(